### PR TITLE
show bar chart in Statistics panel at click

### PR DESCRIPTION
**Pull request by haoxusci**

### CODE REVIEW

File: src/SingleCellAnnotator.imjoy.html [hunk 1]
```diff
@@ -1457,8 +1495,12 @@
               </div>  
               <div v-for="(v, task_name) in s" :key="task_name">
                 <div class="card-subtitle text-gray">{{task_name}}</div>
+                <div :id="k + task_name">
```
Comment (haoxusci) [inline src/SingleCellAnnotator.imjoy.html:1498]: Note you have to click to show the charts. I cannot find a way to automatically execute the fun and show the chart directly when loading

File: src/SingleCellAnnotator.imjoy.html [hunk 2]
```diff
@@ -1457,8 +1495,12 @@
...
+                  <span><button title="show chart" @click="barChartShow(v, k + task_name)">chart</button></span>
+                </div>
```
Comment (haoxusci) [inline src/SingleCellAnnotator.imjoy.html]: Note you need to click to show the charts. pls suggest a way to auto run the func and show charts in this part

**Comment by oeway** [comment]
For the chart library, I would recomend: https://apexcharts.com/
You can find an example here: https://github.com/imjoy-team/imjoy-plugins/blob/master/repository/Im2Im-Dashboard.imjoy.html
Search `apexchart` in the file and you will see how it works.

**Comment by haoxusci** [comment]
> For the chart library, I would recomend: https://apexcharts.com/
> You can find an example here: https://github.com/imjoy-team/imjoy-plugins/blob/master/repository/Im2Im-Dashboard.imjoy.html
> Search `apexchart` in the file and you will see how it works.

why not google? if it works similarly

**Comment by haoxusci** [comment]
> For the chart library, I would recomend: https://apexcharts.com/
> You can find an example here: https://github.com/imjoy-team/imjoy-plugins/blob/master/repository/Im2Im-Dashboard.imjoy.html
> Search `apexchart` in the file and you will see how it works.

had a look at the link you provided. for me, it is more complex than google to learn to use. thus it will take some time (maybe a day) to implement. I would prefer to use google as current if it works good

**Comment by oeway** [comment]
I think they are similar, but if you use the vuejs version of apexchart, you don't need to call plot chart function manually, you only need to bind your data to the chart element. Essentially, you can plot a chart automatically without calling any js function. I recommend it because you are asking how to plot them automatically in one of your comment.

Without vuejs, its actually pretty easy with apexchart: https://apexcharts.com/docs/creating-first-javascript-chart/

Also for the appearance, apexchart is more modern and responsive than google chart.

**Pull request merged**